config/theme.go

```diff
@@ -329,13 +366,16 @@ func (t *Theme) ComputeCache() {
 // These fields should never be referenced in the printers.
 // Instead, they should use the more specific fields, such as [ThemeApply.Created]
 type ThemeBase struct {
+	Added     color.Color // general color for when things are added in
+	Danger    color.Color // general color for when things are bad
 	Info      color.Color // general color for when things are informational
+	Muted     color.Color // general color for when things are less relevant
 	Primary   color.Color // general color for when things are focus
+	Removed   color.Color // general color for when things are removed
 	Secondary color.Color // general color for when things are secondary focus
 	Success   color.Color // general color for when things are good
+	Unchanged color.Color // general color for when things are unchanged
```

Comment:
Okay, I think a bit of your comment. First, I don't agree with you because I'm not fine to use Success for added lines in kubectl diff command. But now I think I know the intention of your thoughts and I agree with you.